pyproject.toml

```diff
@@ -18,39 +18,40 @@ classifiers = [
         'Operating System :: MacOS :: MacOS X',
         'Operating System :: Unix',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.6',
-        'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
         'Programming Language :: Python :: 3.9',
+        'Programming Language :: Python :: 3.10',
+        'Programming Language :: Python :: 3.11',
 ]
 
 [tool.poetry.dependencies]
-python = "^3.6"
-numpy = "^1.19.5"
-pillow = "^8.3.2"
-smqtk-core = ">=0.18.0"
+python = "^3.8"
+pillow = ">=9.2.0"
+numpy = ">=1.22"
```

Comment:
Are these updated version lower-bounds required? I.e. could we instead have `>=` inequalities with the previous versions?

Reply:
Pillow:
There were numerous dependabot alerts, this version is the lowest to address them all.

Numpy:
Dependabot alert.

Neither are transitory packages.